exec/java-exec/src/main/codegen/templates/NewValueFunctions.java

```diff
@@ -17,8 +17,14 @@
  */
 <@pp.dropOutputFile />
 
+<#macro reassignHolder>
+        previous.buffer = buf.reallocIfNeeded(length);
+        previous.buffer.setBytes(0, in.buffer, in.start, length);
+        previous.end = length;
+</#macro>
 
-<@pp.changeOutputFile name="/org/apache/drill/exec/expr/fn/impl/GNewValueFunctions.java" />
+
+<@pp.changeOutputFile name="/org/apache/drill/exeBasec/expr/fn/impl/GNewValueFunctions.java" />
```

Comment:
Is this the correct output destination ?  I get an error in my IDE:
The declared package "org.apache.drill.exec.expr.fn.impl" does not match the expected package "org.apache.drill.exeBasec.expr.fn.impl"	GNewValueFunctions.java	/drill-java-exec/target/generated-sources/org/apache/drill/exeBasec/expr/fn/impl